solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -73,10 +72,7 @@
 public class SolrDispatchFilter extends BaseSolrFilter implements PathExcluder {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  // TODO: see if we can get rid of the holder here (Servlet spec actually guarantees
-  // ContextListeners run before filter init, but JettySolrRunner that we use for tests is
-  // complicated)
-  private ServiceHolder coreService;
```

Comment:
good riddance

Reply:
This was the sticky bit. Before pushing I suggest you beast the tests for a good while. My memory is that without this I got a low level of failures and or deadlocks. The current state while apparently not perfect had a low enough frequency that it didn't crop up. (though I don't remember exactly how long I was beasting things either).

Reply:
Can you please recommend the beasting options I should use today/tonight?

I'd like to merge this to main only and then only backport after awhile.